manager/integration/tests/test_ha.py

```diff
@@ -2141,6 +2141,46 @@ def test_auto_remount_with_subpath(client, core_api, storage_class, sts_name, st
     assert expect_md5sum == md5sum
 
 
+@pytest.mark.skip(reason="WAIT_FOR_UPSTREAM_FIX") # NOQA
+@pytest.mark.skipif( "k3s" not in k8s_api_client.status.node_info.kubelet_version, reason='This test is only compaitable with k3s')  # NOQA
+def test_restart_kubelete_when_pod_with_subpath():
+    """
+    Related issue 3080:
+    https://github.com/longhorn/longhorn/issues/3080
+
+    Steps:
+    0. Check if the cluster runs with k3s
+    1. Create deployment with subpath with pods on the same targeting-node
+    2. Wait for pods become Running
```

Comment:
3. Write some data in the volume, so that we can check the data is intact after the kubelet restart.